SqrlForNet/SqrlCommandWorker.cs

```diff
@@ -653,23 +653,25 @@ public void HelperJson()
 
             if (Options.OtherAuthenticationPaths != null && Options.OtherAuthenticationPaths.Any())
             {
-                responseMessage.Append("\"OtherUrls\":[");
-
+                responseMessage.Append(",\"OtherUrls\":[");
+                int ct = 0;
                 foreach (var optionsOtherAuthenticationPath in Options.OtherAuthenticationPaths)
                 {
                     var xParam = optionsOtherAuthenticationPath.AuthenticateSeparately ? "x=" + (optionsOtherAuthenticationPath.Path.Value.Length) + "&" : string.Empty;
                     var otherUrl = $"sqrl://{Request.Host}{optionsOtherAuthenticationPath.Path}?{xParam}nut={nut}";
                     var otherCheckUrl = $"{Request.Scheme}://{Request.Host}{optionsOtherAuthenticationPath.Path}?check=" + nut;
                     var otherRedirectUrl = $"{Request.Scheme}://{Request.Host}{optionsOtherAuthenticationPath.RedirectToPath}";
                     var otherCancelUrl = Base64UrlTextEncoder.Encode(Encoding.ASCII.GetBytes($"{Request.Scheme}://{Request.Host}{optionsOtherAuthenticationPath.Path}"));
-
+                    if(ct>0)
+                        responseMessage.Append(",");
```

Comment:
Can you add {} around this block to be in line with the rest of the code base and improve readability

Reply:
Made changes and pushed